.github/workflows/coverity.yml

```diff
@@ -37,8 +37,8 @@ jobs:
     env:
       CC: gcc
       CXX: g++
-      COVERITY_TOKEN: AK2euT-neBcvZB1g_m3pbg
       MAKEFLAGS: j4
+      TOKEN: qjcM1CWLcq9PJB3yL0ZXIw
```

Comment:
@maxirmx I believe we don't need the Coverity token when the repository is public. Can you confirm?

Reply:
@ronaldtse 
It does not work without token: https://github.com/rnpgp/sexpp/actions/runs/5375142016/jobs/9751251719
Documentation is somewhat unclear about it but I found an action https://github.com/marketplace/actions/unofficial-coverity-scan and its documentation says that the TOKEN is required

I am using this action now.  We may want to move Coverity token to secrets and use ribose e-mail instead of my personal.